tools/integration_tests/integration_test_utils.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I think we try to avoid dumping grounds like utils.py or helpers.py, cc @dkgi 
Also, why repeating `integration_test_` when it's already the directory name?
I would just call it `utils.py` or `library.py`. Open to better suggestions.

Reply:
> I think we try to avoid dumping grounds like utils.py or helpers.py

It contains functions common to the internal tests and the open source tests. I've been asked to reformat the single file I had to prevent duplicate code. That's why we have two file.
Quote @r0rshark:
>  I plan to share this code across Opensource and our internal integration tests. What do you think should be a good place for this to live? Something like pyre-check/tools/integration_tests ? cc @gbleaney who mentioned a possible refactoring that might solve the "file location" problem

I will just rename it to utils.py!

Reply:
Yes I would like to avoid duplicating the integration tests code yet another time